pkg/config/config.go

```diff
@@ -129,7 +124,7 @@ func (c *Dumper) WriteFiles() (err error) {
 func getAppendOrInsertConfigData(path string, data []byte, insert bool) ([]byte, error) {
 	var configs [][]byte
 	context, err := os.ReadFile(filepath.Clean(path))
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
ignore `NotExist` error so Dumper can create a new file later.

Reply:
seems if os.IsNotExist(err) {create file } is better?

Reply:
not need to create file manually here, file will be created later in `file.WriteFile`